src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
+    return <PersonLink person={person} people={people} key={person.slug} />;
```

Comment:
Passing the entire 'people' array to each 'PersonLink' might not be necessary. If 'PersonLink' only needs the 'person' prop, consider removing the 'people' prop to avoid passing unnecessary data.